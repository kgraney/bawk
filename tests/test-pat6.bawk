
/* these values should be different */

/000a value:int4/  { print(value); }
/000a value:uint4/ { print(value); }

/000a value:int2/ { print (value); }
/000a value:uint2/ { print (value); }

/000a value:int1/ { print (value); }
/000a value:uint1/ { print (value); }


/* these values should be the same */

/000d value:int4/  { print(value); }
/000d value:uint4/ { print(value); }

/000d value:int2/  { print(value); }
/000d value:uint2/ { print(value); }

/000d value:int1/  { print(value); }
/000d value:uint1/ { print(value); }
