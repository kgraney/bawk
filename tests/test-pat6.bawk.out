-268435446
4026531850
-4096
61440
-16
240
1229472850
1229472850
18760
18760
73
73
